models/intermediate/int_jira__issue_sprint.sql

```diff
@@ -19,15 +19,17 @@ sprint_field_history as (
 
     select 
         field_history.*,
+        sprint.*,
```

Comment:
Is this needed? Or did we just add it for testing purposes?

Reply:
I think it is needed as lines 58-62 reference the sprint fields, but would you be able to confirm this for me.

Reply:
Yes that's correct! Line 22 is needed because I moved the `sprint` table join up to the `sprint_field_history` CTE.

Reply:
Sounds good, that's what I was thinking but wanted to confirm on your end!